staging/src/k8s.io/kubectl/pkg/util/resource/resource.go

```diff
@@ -25,6 +25,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/util/sets"
+	helpers "k8s.io/component-helpers/resource"
```

Comment:
FYI, I just commented on https://github.com/kubernetes/kubernetes/issues/124537 - we may be able to just remove this file instead.

Reply:
ack